test/fabrik.DirectorManager.spec.js

```diff
@@ -55,6 +58,26 @@ describe('fabrik', function () {
         manager.findNetworkSegmentIndex(used_guid).then(res => expect(res).to.eql(21));
       });
     });
-
+    describe('#isRestorePossible', function () {
+      it('should return false when plan not in restore_predecessors', function () {
+        // restore not possible from small to xsmall
+        manager = new DirectorManager(catalog.getPlan(xsmall_plan_id));
+        manager.update_predecessors = [];
+        killJsHintWarning = expect(manager.isRestorePossible(small_plan_id)).to.be.false;
```

Comment:
what is this variable ?

`killJsHintWarning` ?

Reply:
Jshint expects a assignment or function call in each statement and not an expression.
The expect check for true/false is a expression and not a function call like other expect statements.
So `killJsHintWarning` is used to make the expression into and assignment. 
Also renamed `killJsHintWarning` to `return_value`.

Reply:
Ok , got it .